packages/url-loader/src/constants/parameters.ts

```diff
@@ -143,9 +143,31 @@ export const format = {
 
 /** Gravity */
 
-const gravitySchema = z.string();
+export type Gravity = z.infer<typeof gravitySchema>;
+
+const gravitySchema = z.union([
+  z.enum([
+    "auto",
+    "auto_content_aware",
+    "center",
+    "custom",
+    "east",
+    "face",
+    "face_center",
+    "multi_face",
+    "north",
+    "north_east",
+    "north_west",
+    "south",
+    "south_east",
+    "south_west",
+    "west"
+  ]),
+  // Quirk to allow enum + string
+  z.intersection(z.string(), z.object({}))
+]);
 
-export const gravity = {
+export const c = {
```

Comment:
hey just curious before i fix this, was this an accidental change?

Reply:
idk how tests didnt run and catch htis 🤔

Reply:
Oops. That was me trying to run a git command probably and I didn't realize that I made that change, but defintely odd it didn't get caught.

Reply:
Fixed in b90ea5d0bf94d3b2172124f6d7bd0fad5d5230b3

Reply:
good idea, not sure why i didnt add that i usually have that sorta thing, but i also guess iw ould have expected that to be caught in tests. weird

Reply:
Typically we'll approve the workflow awaiting approval if the PR doesn't look sketch.

![CleanShot 2024-08-26 at 12 34 17](https://github.com/user-attachments/assets/b7d1213f-c76d-495a-bc0d-0e9050bbd755)

Does the release action only runs when code is pushed to main though? If that's the case, consider running the build, linting or whatever else you consider CI worthy and let that run on PRs so you catch it before a release.

Reply:
> good idea, not sure why i didnt add that i usually have that sorta thing, but i also guess iw ould have expected that to be caught in tests. weird

But yeah weird the tests didn't catch that, or do tests not run on PRs and only release?

Reply:
im not seeing the approval option in my UI... in not sure what im not getting it

Reply:
its a test and release action, the first step is for testing, the 2nd step is optionally releasing